.gitignore

```diff
@@ -37,11 +36,6 @@ __pycache__/
 /src/libcore/unicode/SpecialCasing.txt
 /src/libcore/unicode/UnicodeData.txt
 /src/libcore/unicode/downloaded
-/stage[0-9]+/
-/target
-target/
```

Comment:
One of this or `/target` may be worth keeping since if you're playing around with just a bare `cargo build` it'll ignore these directories by default

Reply:
Okay, I re-added `/target` next to `/build`.